quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java

```diff
@@ -102,16 +102,6 @@ void testRuntimePropFromQuarkusPropsIsAppliedWithoutRebuild(LaunchResult result)
         cliResult.assertNoBuild();
     }
 
-    @Test
-    @BeforeStartDistribution(UpdateHibernateMetricsFromQuarkusProps.class)
-    @Launch({ "start", "--http-enabled=true", "--hostname-strict=false" })
-    @Order(7)
-    void testBuildRunTimeMismatchOnQuarkusBuildPropWarning(LaunchResult result) {
```

Comment:
This test makes no sense as it is basically checking messages from Quarkus.

There are more tests here we can review/remove but it is out of the scope of this PR.